docs/tutorials/create-a-service-to-detect-anomalies-using-a-model-built-from-scratch.md

```diff
@@ -169,9 +169,27 @@ Create a freeze file to pin all dependencies to their current versions:
 pip freeze --local --all > requirements-all.txt
 ```
 
+Finally, copy the line
```

Comment:
Wouldn't it be better to let the user choose if he wants to stay up to date or if he wants his service to be consistent over time. To do so tell him to either keep the line in requirements.txt or in requirements-all.txt. One for the updates and one for consistency ?

Reply:
Is this better?:

```
The specific `common-code @ git+https://github.com/swiss-ai-center/common-code.git@<commit>`
line will conflict with the more general line in `requirements.txt` due to the explicit commit reference.

From there, you have two options:
* **Easier Update:** Remove the specific `common-code @ git+https://github.com/swiss-ai-center/common-code.git@<commit>` line from `requirements-all.txt`. This allows for easier updates of the common-code dependency without adjusting service dependencies.
* **Consistent Dependencies:** Remove the generic line in `requirements.txt` to keep dependencies consistent across machines and time. This will ensure that the same versions of the dependencies are installed on
every machine if you ever share your code with someone else.
```

Reply:
Yes, this is more explicit !